eslint.config.js

```diff
@@ -29,6 +29,7 @@ export default [
       ...react.configs['jsx-runtime'].rules,
       ...reactHooks.configs.recommended.rules,
       'react/jsx-no-target-blank': 'off',
+      'react/prop-types': 'off',
```

Comment:
You can also set this to `warn` instead of `off` so all you get it a warning and not a full-on error so when we get to the lesson on prop types, you'd still get feedback from the linter.